packages/cardano-services/package.json

```diff
@@ -17,25 +17,27 @@
   "license": "MPL-2.0",
   "scripts": {
     "build": "tsc --build ./src && bash ./copy-openApi.sh",
-    "tscNoEmit": "shx echo typescript --noEmit command not implemented yet",
     "cleanup": "shx rm -rf dist node_modules",
+    "cli": "ts-node --transpile-only src/cli.ts",
+    "coverage": "yarn test --coverage",
     "lint": "eslint --ignore-path ../../.eslintignore \"**/*.ts\"",
     "lint:fix": "eslint --fix --ignore-path ../../.eslintignore \"**/*.ts\"",
-    "test": "jest -c ./jest.config.js",
-    "test:e2e": "shx echo 'test:e2e' command not implemented yet",
-    "coverage": "yarn test --coverage",
     "prepack": "yarn build",
     "pretest": "yarn build",
+    "test": "jest -c ./jest.config.js",
+    "test:e2e": "shx echo 'test:e2e' command not implemented yet",
     "test:debug": "DEBUG=true yarn test",
+    "test:load": "jest -c ./load.jest.config.js --runInBand",
+    "tscNoEmit": "shx echo typescript --noEmit command not implemented yet",
     "run:http-server": "ts-node --transpile-only src/run.ts",
     "run:http-server:debug": "npx nodemon --legacy-watch --exec 'node -r ts-node/register --inspect=0.0.0.0:9229 src/run.ts'",
     "run:tx-worker": "ts-node --transpile-only src/startWorker.ts",
-    "run:tx-worker:debug": "npx nodemon --legacy-watch --exec 'node -r ts-node/register --inspect=0.0.0.0:9229 src/startWorker.ts'",
-    "cli": "ts-node --transpile-only src/cli.ts"
+    "run:tx-worker:debug": "npx nodemon --legacy-watch --exec 'node -r ts-node/register --inspect=0.0.0.0:9229 src/startWorker.ts'"
   },
   "devDependencies": {
     "@cardano-sdk/cardano-services-client": "0.2.0",
     "@cardano-sdk/util-dev": "0.2.0",
+    "@cardano-sdk/wallet": "0.2.0",
```

Comment:
Yes, a new package makes sense. We can share the config and also present the SDK features from a single source.

Reply:
Sorry @mkazlauskas and @rhyslbw:
do you mean a new package dedicated to this test?

Reply:
Sorry @mkazlauskas , which ticket did you updated? I can't see any change to https://input-output.atlassian.net/browse/ADP-1828 (the ticket relative to this PR)

Reply:
ADP-1759 - you can keep the dependency in this PR - nothing to do.

Since this repo is public and tickets are private, please reference the tickets without the link (just the ID).